UI/payments/payment2.html

```diff
@@ -162,7 +162,7 @@
                                  data-dojo-type="dijit/form/Button">
```

Comment:
Apparently, given this line, that bit didn't end up either on master or on the webpack branch.

Reply:
It seems it was indeed missing on master. Fixed.